src/communication/azext_communication/version.py

```diff
@@ -1,11 +0,0 @@
-# --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
-
-
-VERSION = '1.11.2'
-
-
-def cli_application_id():
```

Comment:
[from ..version import cli_application_id](https://github.com/Azure/azure-cli-extensions/blob/main/src/communication/azext_communication/manual/_client_factory.py#L9)
Can we just delete the relevant code in setup.py to prevent causing any breaking change?